tests/index.js

```diff
@@ -98,29 +103,44 @@ tap('working with dats', function (t) {
     .then(() => app.client.click('button.confirm-button'))
     .then(() => wait())
     .then(() => app.client.getText('.tutorial'))
-    .then((text) => t.ok(text.toLowerCase().match(/share/), 'now the dat is gone and welcome screen is back'))
+    .then(text =>
+      t.ok(
+        text.toLowerCase().match(/share/),
+        'now the dat is gone and welcome screen is back'
+      )
+    )
     .then(() => endTest(app))
 })
 
 // Create a new app instance
 function createApp () {
   return new spectron.Application({
     path: path.join(__dirname, '../node_modules/.bin/electron'),
-    args: [path.join(__dirname, '../index.js'), '--data', TEST_DATA, '--db', TEST_DATA_DB],
+    args: [
```

Comment:
Is it just me or has this become significantly less readable.

Reply:
yes, it's not perfect every single time, but it's consistent

Reply:
before, this line was 91 chars long, limiting lines to 80 chars is a common practice